src/plugins/intel_cpu/src/emitters/snippets/aarch64/kernel_executors/gemm_copy_b.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "common/utils.hpp"
+#include "cpu_memory.h"
+#include "emitters/snippets/brgemm_generic.hpp"
+#include "emitters/utils.hpp"
+#include "kai/ukernels/matmul/matmul_clamp_f32_f32_f32p/kai_matmul_clamp_f32_f32_f32p8x1biasf32_6x8x4_neon_mla.h"
+#include "kai/ukernels/matmul/matmul_clamp_f32_f32_f32p/kai_matmul_clamp_f32_f32_f32p_interface.h"
+#include "kai/ukernels/matmul/pack/kai_rhs_pack_kxn_f32p8x1biasf32_f32_f32_neon.h"
+
+namespace ov::intel_cpu::aarch64 {
+
+struct GemmCopyBKernelKaiConfig : public snippets::KernelExecutorBase::GenericConfig {
+public:
+    GemmCopyBKernelKaiConfig() = default;
+    GemmCopyBKernelKaiConfig(const size_t N, const size_t K, const size_t n_blk_size);
+
+    bool operator==(const GemmCopyBKernelKaiConfig& rhs) const;
+    bool operator!=(const GemmCopyBKernelKaiConfig& rhs) const {
+        return !(*this == rhs);
+    }
+
+    [[nodiscard]] std::unique_ptr<GenericConfig> get_clone_ptr() const override {
+        return std::make_unique<GemmCopyBKernelKaiConfig>(*this);
+    }
+
+    [[nodiscard]] bool is_completed() const override;
+
+#ifdef SNIPPETS_DEBUG_CAPS
+    virtual std::string to_string() const override;
+#endif
+
+    void update(size_t N, size_t K, size_t n);
+
+    size_t hash() const override {
+        return m_hash;
+    }
+
+    [[nodiscard]] size_t get_N() const {
+        return m_N;
+    }
+    [[nodiscard]] size_t get_K() const {
+        return m_K;
+    }
+    [[nodiscard]] size_t get_n() const {
```

Comment:
We definitely should rename it - two methods with the similar namings. What's about `get_n_blk_size` or `get_wei_n_blk`?

The same comment for method `update` above

Reply:
`get_n_blk_size `is used now. thanks Alexandra!